addon/lib/document.js

```diff
@@ -8,7 +8,10 @@ import { decodeId } from "ember-caluma/helpers/decode-id";
 import Base from "ember-caluma/lib/base";
 import { intersects } from "ember-caluma/utils/jexl";
 
-const onlyNumbers = (nums) => nums.filter((num) => !isNaN(num));
+const onlyNumbers = (nums) =>
+  nums.filter(
+    (num) => Number.isInteger(num) || (Number(num) === num && num % 1 !== 0)
```

Comment:
Uhm, how about `(num) => typeof num === "number"`? I thought we agreed that we don't want to cast the input.

Reply:
It don't know exactly what the right part of the OR does, but it doesn't allow strings. My test above yielded this result: `[ 1, 2, 1.2, 1 ]`

Reply:
As discussed with @kaldras, I've updated it to `(num) => !isNaN(num) && typeof num === "number"` since `typeof NaN === "number" results in `true``